src/apiCalls.js

```diff
@@ -17,18 +19,18 @@ export const userPantry = (pantryUpdate) => {
   fetch('http://localhost:3001/api/v1/users', {
     method: 'POST',
     headers: { 'Content-Type': 'application/json' },
-    body: JSON.stringify(pantryUpdate)
+    body: JSON.stringify(pantryUpdate),
   })
     .then(response => response.json())
-    .then(ingred => console.log(`success: ${ingred}`))
```

Comment:
Writing this particular part of the code for the fetch post process was really helpful for me.  I hope to gain more understanding of the process as we work on our individual projects.